hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/AbiConstants.java

```diff
@@ -131,4 +131,10 @@ private AbiConstants() {
     public static final int ABI_ID_GET_TOKEN_INFO = 0x1f69565f;
     // getNonFungibleTokenInfo(address token, int64 serialNumber)
     public static final int ABI_ID_GET_NON_FUNGIBLE_TOKEN_INFO = 0x287e1da8;
+
+    // getTokenDefaultFreezeStatus(address)
+    public static final int ABI_ID_GET_TOKEN_DEFAULT_FREEZE_STATUS = 0xa7daa18d;
+
+    // getTokenDefaultKycStatus(address)
+    public static final int ABI_ID_GET_TOKEN_DEFAULT_KYC_STATUS = 0x335e04c1;
```

Comment:
You can define variable name to the signature (e.g. tokenId) to be compliant with the rest of the comments.

Reply:
DONE